### PR TITLE
fix(w3c/linter-rules/required-sections): support other locales

### PR DESCRIPTION
**Pull request by marcoscaceres**
Forgot that this would get upset if the document was not in English.

### CODE REVIEW

File: src/w3c/linter-rules/required-sections.js [hunk 1]
```diff
@@ -40,15 +55,20 @@ export function run(conf) {
     return;
   }
 
+  // We can't check for headers unless we also have a translation
+  if (!localizationStrings[lang]) {
+    return;
```
Comment (sidvishnoi) [inline src/w3c/linter-rules/required-sections.js:60]: Should add a warning here. A `console.warn` if not `showWarning`..

File: src/w3c/linter-rules/required-sections.js [hunk 2]
```diff
@@ -40,15 +55,20 @@ export function run(conf) {
     return;
   }
 
+  // We can't check for headers unless we also have a translation
+  if (!localizationStrings[lang]) {
```
Comment (sidvishnoi) [inline src/w3c/linter-rules/required-sections.js:59]: There's a little caveat here. I think we should normalize the `lang` here like [`getIntlData` does](https://github.com/w3c/respec/blob/0067ad45facb56565463a5fe18b8eb97ffc222a2/src/core/utils.js#L181).

File: src/w3c/linter-rules/required-sections.js (hunk 1 above)
Comment (marcoscaceres) [reply src/w3c/linter-rules/required-sections.js:60]: Not sure about that ... there is nothing the spec Editor can do about it? 

Also, spec translations are not canonical, so the warnings is just nice to have in this context.

File: src/w3c/linter-rules/required-sections.js (hunk 2 above)
Comment (marcoscaceres) [reply src/w3c/linter-rules/required-sections.js:59]: yeah, can run it over the same code.

File: src/w3c/linter-rules/required-sections.js (hunk 1 above)
Comment (marcoscaceres) [reply src/w3c/linter-rules/required-sections.js:62]: Went with the  console.warn ... I think that works for now.

Comment (sidvishnoi) [reply src/w3c/linter-rules/required-sections.js:62]: Yeah, console.warn is good. As the author (or us) will know what's happening if that happens. Less unexpected.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**